src/nl/hannahsten/texifyidea/util/Files.kt

```diff
@@ -163,7 +163,8 @@ fun PsiFile.findRootFile(): PsiFile {
             return@forEach
         }
 
-        if (file.isIncludedBy(file, inclusions)) {
```

Comment:
also not sure about this, given that findRootFile is called all the time, and that for the past years. Hope I didn't break something  important